### ISSUE TITLE
Help with authentication?

### ISSUE THREAD

**Issue by AllenDowney** [ericmjl/llamabot#100]
This tool looks super useful, so thanks!

I am just getting set up and I'm running into this error:

```
NotFoundError: litellm.NotFoundError: OpenAIException - Error code: 404 - {'error': {'message': 'The model 
`gpt-4-turbo` does not exist or you do not have access to it.', 'type': 'invalid_request_error', 'param': None, 
'code': 'model_not_found'}}
```

I have an environment variable with my OpenAI API key:

```
$ echo $OPENAI_API_KEY
sk-proj-etcetc
```

But I don't think the API key is working. I also tried creating a `SimpleBot`, and I am also getting authentication errors.

So I could use some help debugging. Is there a simple test I can use to check that my API key is working or figure out why it's not?

**Comment by sivagowrisankar** [comment]
Use a curl script to check if you can connect to a model with the API KEY.

curl https://api.openai.com/v1/chat/completions \
  -H "Content-Type: application/json" \
  -H "Authorization: Bearer $OPENAI_API_KEY" \
  -d '{
    "model": "gpt-4o",
    "messages": [
      {
        "role": "system",
        "content": "You are a helpful assistant."
      },
      {
        "role": "user",
        "content": "Hello!"
      }
    ]
  }'

**Comment by ericmjl** [comment]
@AllenDowney we've resolved this offline, but commenting here for others who might find this issue via Google. 

It looks like the API key may be associated with an account that doesn't yet have access to those models? Definitely, @sivagowrisankar's response helps disambiguate what models one has access to. Also, I went digging through the docs, and I guess if one is on the Free tier (i.e. with no credit card assigned to your account), it might be the case that `gpt-4-turbo` is not accessible. (I referenced this docs: https://platform.openai.com/docs/guides/rate-limits/usage-tiers?context=tier-free)

Hope this helps!
